### ISSUE TITLE
Undeclared configure_requires dependency File::Find::Rule

### ISSUE THREAD

**Issue by eserte** [shawnlaffan/perl-alien-geos#2]
On some of my smokers:
```
Output from '/usr/local/bin/perl Makefile.PL':

Can't locate File/Find/Rule.pm in @INC (you may need to install the File::Find::Rule module) (@INC contains: ...
```

Probably this module has to be mentioned in configure_requires.

**Comment by shawnlaffan** [comment]
Thanks for reporting.  

This should be fixed by 51740fac3071f556803fad01ab52ae688e39d830

A new version has been uploaded to CPAN.

**Comment by shawnlaffan** [comment]
Mark as fixed.  CPAN testers is green for the relevant systems.
